docs/integrations.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: default
+title: Integrations
+has_children: false
+has_toc: false
+nav_order: 9
+---
+
+# Integration with SmartClean Matrix
+_Integrations allow you to set up access to data and events from our platform in a standard way._
+
+## Integration mechanisms supported
+1. Receive data and events from our platform on your web service.
+2. Query certain information from our platform by making web requests.
+3. Query data and events from an SQL database.
+4. Query data from our platform by using our SDK for certain programming languages.
```

Comment:
5. Through a federated access mechanism using OAuth client credentials or authorization code grant.

Reply:
1. Need help to understand what to put in description for this point.
(The remaining 4 points have been described in a section in this page)
2. This line seems to be referring to point 2 (access by making web requests, i.e. the API page)
     - If it is separate from the other 4 options, then need to describe how users will access this option